conf/application.conf

```diff
@@ -109,7 +109,7 @@ microservice {
         max-uploads = 4
         min-file-size = 0
         max-file-size = 5242880
-        upscan-store-expiry-time = 7
+        upscan-store-expiry-time = 7days
```

Comment:
are you converting this to seconds somewhere? also should there be a space between `7` and `days`?

Reply:
I put no space in between to highlight the fact that we can rite it like that so we're more confident it works without spaces in app-config-* where we can't have whitepace.
In the TaskListController it's being converted to seconds at one point on class start so we can pass it into `.plusSeconds` for the expiry stuff